[Integrations] Testcontainers, Kubernetes Operator, Kratix (#4146)
dapr/docs: daprdocs/content/en/developing-applications/integrations/Diagrid/test-containers.md

---
type: docs

> title: "How to: Integrate using Diagrid's Testcontainers Dapr Module"

Reviewer: ```suggestion
title: "How to: Integrate using Testcontainers Dapr Module"
```

> linkTitle: "Diagrid Testcontainers"

Reviewer: ```suggestion
linkTitle: "Dapr Testcontainers"
```

weight: 3000
description: "Use the Dapr Testcontainer module from your Java application"
---

> You can use the Testcontainers Dapr Module provided by Diagrid to set up Dapr locally in your Java applications. Simply add the following dependency to your Mave project:

Reviewer: ```suggestion
You can use the Testcontainers Dapr Module provided by Diagrid to set up Dapr locally for your Java applications. Simply add the following dependency to your Maven project:
```
